AAI/AAIConnectProfile.md

```diff
@@ -346,47 +320,53 @@ upstream.
 
     2.  The `exp` must not exceed 30 days after the `iat`.
 
+    3.  The `scope` claim, if included, MUST NOT contain "openid" as a
+        space-delimited substring.
+
 3.  If the Embedded Token body contains the "openid" scope:
 
     1.  The token MUST conform with the specification for access tokens.
-        The Claim Signatory MUST meet conformance requirements of an
-        Identity Broker as it pertains to Embedded Tokens with "openid"
-        scope.
+        The Embedded Token Signatory MUST meet conformance requirements of an
+        Broker as it pertains to Embedded Tokens with "openid" scope.
           
-    2.  If the `exp` exceeds the `iat` by more than 1 hour, the Claim
-        Signatory MUST provide an introspection_endpoint and publish its URI
-        within its Discovery service endpoint. The introspection endpoint
-        MUST be able to support returning the revocation status of the
-        Embedded Token.
+    2.  If the `exp` exceeds the `iat` by more than 1 hour, the Embedded
+        Token Signatory MUST provide an introspection_endpoint and publish
```

Comment:
This is probably too big extra requirement for Embedded Token Signatory. In practice, the signatories (like EGA) would need to run their own OIDC provider which we have tried to avoid. Also all clients querying an introspection_endpoint must have the client credentials to make the query.

Reply:
Also an introspection endpoint provides info only about access tokens, not about arbitrary JWT tokens. Thus this part of the specification does not work for Embedded Tokens which would not be access tokens.